packages/design-system-docs/src/pages/components/DateField/datefield.example.html

```diff
@@ -108,7 +47,7 @@
           ><span>Year</span></label
         >
         <input
-          class="ds-c-field ds-c-field--error ds-c-field--year"
+          class="ds-c-field ds-c-field--year"
```

Comment:
```suggestion
          class="ds-c-field ds-c-field--error ds-c-field--year"
```
keep year field consistent with the inverse example